src/orbital/azext_orbital/azext_metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+    "azext.isExperimental": true,
```

Comment:
Do you need to release this extension in experimental? recommend to mark this extension isPreview first

Reply:
We are releasing the extension for our GA. What should we mark the extension in that case?

Reply:
> We are releasing the extension for our GA. What should we mark the extension in that case?

In that case, you can remove this line.